presto-main/src/main/resources/webapp/assets/utils.js

```diff
@@ -304,6 +304,16 @@ function formatState(state, fullyBlocked) {
     }
 }
 
+function getFirstParameter(searchString) {
+    var searchText = searchString.substring(1);
```

Comment:
It's not clear what the behavior should be if it's not '?', so I would prefer that it fail.